pkg/irreparabledb/dbx/irreparabledb.dbx

```diff
@@ -0,0 +1,24 @@
+// dbx.v1 golang irreparabledb.dbx .
+model irreparabledb (
+	key segmentpath
+
+	field segmentpath blob
+	field segmentdetail blob (updatable)
+	field pieces_lost_count int64 (updatable)
+	field seg_damaged_unix_sec int64 (updatable)
+	field repair_attempt_count int64 (updatable)
+)
+
+create irreparabledb ( )
```

Comment:
Remove things here that are not being used.

Reply:
done